specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/applications.json

```diff
@@ -250,6 +250,14 @@
                     "type": "integer",
                     "format": "int32",
                     "description": "The public port to connect to."
+                },
+                "subDomainSuffix":{
+                    "type": "string",
+                    "description": "The subDomainSuffix of the application and can not greater than 3 characters."
+                },
+                "disableGatewayAuth":{
+                    "type": "boolean",
```

Comment:
FYI ARM team advises against boolean properties in specs, it will be a breaking change if you ever decide to add a third value and make it a string.

Reply:
> FYI ARM team advises against boolean properties in specs, it will be a breaking change if you ever decide to add a third value and make it a string.

Thanks for your reminder. It makes sense to me. We think it should be boolean.